src/FixedDataTable.js

```diff
@@ -742,7 +742,7 @@ var FixedDataTable = React.createClass({
   _onColumnReorderMove(
     /*number*/ deltaX
   ) {
-    var reorderingData = this.state.columnReorderingData;
+    var reorderingData = JSON.parse(JSON.stringify(this.state.columnReorderingData));
```

Comment:
Ah cool, instead of a TODO just leave a comment saying you do this to deep clone the object.